src/manifest.json

```diff
@@ -21,6 +21,7 @@
       "css": ["content.styles.css"]
     }
   ],
+  "host_permissions": ["http://*/"],
```

Comment:
Can you please explain why is this needed?

Reply:
The `host_permissions` are required if you want to inject something in a matching page, download from a matching URL, or observe via webRequest and similar APIs.

Most extensions will need it, not all of course, I spent a considerable amount of time trying to figure out why things were not working and that was one of the issues.

Could be added to the readme or removed, as you wish, personally, for a boilerplate, I would add all the possible fields in there with default values so new developers know what they can use or could be the cause of an issue.

Reply:
It might make sense to have the fields you *may* want (as in my case, including `host_permissions`) commented out in the file, and in the uncommented code support the most basic and least permissive cases. It's an approach I've seen in other config files that would also work here, and also one that doesn't risk someone accidentally asking for permissions they don't need.